src/ResourceManager/IotHub/Commands.IotHub/Commands.IotHub.csproj

```diff
@@ -178,6 +179,10 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Microsoft.Azure.Commands.IotHub.dll-Help.xml">
```

Comment:
Remove.  Follow the directions here for creating help markdowns:  https://github.com/Azure/azure-powershell/blob/preview/documentation/help-generation.md